src/Microsoft.AspNetCore.Server.Kestrel/Filter/Internal/LibuvStream.cs

```diff
@@ -5,10 +5,10 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Server.Kestrel.Http;
-using Microsoft.AspNetCore.Server.Kestrel.Infrastructure;
+using Microsoft.AspNetCore.Server.Kestrel.Internal.Http;
+using Microsoft.AspNetCore.Server.Kestrel.Internal.Infrastructure;
 
-namespace Microsoft.AspNetCore.Server.Kestrel.Filter
+namespace Microsoft.AspNetCore.Server.Kestrel.Filter.Internal
 {
     public class LibuvStream : Stream
```

Comment:
`LoggingStream` is already internal as-in-the-keyword, so it probably belongs in the internal namespace as well.